examples/kubernetes/1.10/cilium-ds.yaml

```diff
@@ -35,7 +35,7 @@ spec:
         - name: clean-cilium-state
           image: docker.io/library/busybox:1.28.4
           imagePullPolicy: IfNotPresent
-          command: ['sh', '-c', 'if [ "${CLEAN_CILIUM_STATE}" = "true" ]; then rm -rf /var/run/cilium/state; rm -rf /sys/fs/bpf/tc/globals/cilium_*; fi']
+          command: ['sh', '-c', 'if [ "${CLEAN_CILIUM_STATE}" = "true" ]; then rm -rf /var/run/cilium/state; fi; if [ "${CLEAN_CILIUM_STATE}" = "true" ] || [ "${CLEAN_CILIUM_BPF_STATE}" = "true" ]; then rm -rf /sys/fs/bpf/tc/globals/cilium_* /var/run/cilium/bpffs/tc/globals/cilium_*; fi']
```

Comment:
Should we start thinking about creating a `cilium/cilium-clean` container to run all of these commands?

Reply:
It's starting to get to the point where it would be a bit tidier.